src/main/java/frc/robot/maps/subsystems/ShooterMap.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot.maps.subsystems;
+
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+import com.chopshop166.chopshoplib.logging.LoggableMap;
+import com.chopshop166.chopshoplib.motors.SmartMotorController;
+
+public class ShooterMap implements LoggableMap<ShooterMap.Data> {
+
+    public final SmartMotorController motor;
+
+    public ShooterMap() {
+        this(new SmartMotorController());
+    }
+
+    public ShooterMap(SmartMotorController motor) {
+        this.motor = motor;
+    }
+
+    @Override
+    public void updateData(Data data) {
```

Comment:
This needs to update the other fields. We also need to figure out how to get temp and current from both motors.